src/Sulu/Bundle/CategoryBundle/Controller/CategoryController.php

```diff
@@ -269,8 +274,11 @@ protected function getListRepresentation(
         // generate expressions for collected parent-categories
         $parentExpressions = [];
         foreach ($idsToExpand as $idToExpand) {
+            $fieldDescriptor = $listBuilder->getFieldDescriptor('parent');
+            Assert::notNull($fieldDescriptor, 'There is no field descriptor for field "parent"');
+
             $parentExpressions[] = $listBuilder->createWhereExpression(
-                $listBuilder->getFieldDescriptor('parent'),
+                $fieldDescriptor,
                 $idToExpand,
                 ListBuilderInterface::WHERE_COMPARATOR_EQUAL
             );
```

Comment:
No change in behavior, only making the error message more helpful. Error message before: Expected FieldDescriptorInterface got null